.github/workflows/ci.yml

```
@@ -71,6 +71,9 @@ jobs:
       - run: python scripts/doc_checker.py modin/utils.py
       - run: python scripts/doc_checker.py modin/experimental/sklearn
       - run: python scripts/doc_checker.py scripts/doc_checker.py
+      - run: |
+          python scripts/doc_checker.py modin/backends/pyarrow/query_compiler.py \
+            modin/backends/pyarrow/parsers.py
```

```suggestion
      - run: python scripts/doc_checker.py modin/backends/pyarrow
```
Please, add docstring for `__init__.py` file to make this possible